app/qml/dialogs/MMMigrateToMerginDialog.qml

```diff
@@ -8,22 +8,26 @@
  ***************************************************************************/
 
 import QtQuick
-import QtQuick.Dialogs
 
-MessageDialog {
+import "../components"
+
+MMDrawerDialog {
   id: root
 
   signal migrationRequested()
 
-  title: qsTr( "Upload project to Mergin?" )
-  text: qsTr( "This project is currently not uploaded on Mergin. Upload it to Mergin in order to activate synchronization and collaboration. \nClick 'Yes' to upload it." )
+  picture: __style.positiveMMSymbolImage
+  bigTitle: qsTr( "Upload project to Mergin?" )
+  description: qsTr( "This project is currently not uploaded on Mergin. Upload it to Mergin in order to activate synchronization and collaboration." )
```

Comment:
This project is currently not uploaded on cloud. Upload it in order to activate synchronization and collaboration.